src/sci/impl/parser.cljc

```diff
@@ -57,7 +58,8 @@
                            :read-cond :allow
                            :features features
                            :auto-resolve auto-resolve
-                           :syntax-quote {:resolve-symbol #(fully-qualify ctx %)})
+                           :syntax-quote {:resolve-symbol #(fully-qualify ctx %)}
+                           :tools.reader/opts {:readers readers})
```

Comment:
Not that it matters much, but edamame also supports passing in `:readers` directory.

Reply:
I thought I tried that but I must have been doing something else wrong, just the `:readers` key works. I'll change that and get a unit test tonight.

Reply:
Actually I was wrong, it seems like you cannot just pass 
```
:readers readers
```
In the `parse-opts` instead of 
```
:tools.reader/opts {:readers readers}
```
It seems like `normalize-args` (https://github.com/borkdude/edamame/blob/6e0fcd1bd9767c3fb7f3595ce91828f82ccaf760/src/edamame/impl/parser.cljc#L496) would have to be called on the opt map to allow the abbreviated version. This is done for you in edamame's higher level `parse-string` and `parse-string-all` but not `parse-next` which is being used here. 

Let me know If I'm missing something obvious.  

I think I will also add a check to see if the user actually passes a `:readers` instead of assoc'ing `:tools.reader/opts {:readers nil}` when no readers are provided.

Reply:
I think you're right, I kind of forgot that I was calling a non-public function of edamame. The way you are doing it is fine, let's leave it.